package.json

```diff
@@ -46,5 +47,9 @@
   },
   "publishConfig": {
     "access": "public"
-  }
+  },
+  "peerDependencies": {
```

Comment:
Is there a reason why you choose to use a peerDependency? I don't have a clear rule in my head about when to use peers

Reply:
My thinking here is that this library is not very useful by itself. I expect consumers to implement custom RxJS operators that get used with functions like `handlers.some()`. This means that this code and the consumer code need to agree on a single version of RxJS that they both share. My understanding is that this is the main use case for `peerDependencies`.